CMakeLists.txt

```diff
@@ -24,6 +24,8 @@ set(SRC_COMPONENTS
     ${SRC_DIR}Components/Component.cpp
     ${SRC_DIR}Components/Capture.h
     ${SRC_DIR}Components/Capture.cpp
+    ${SRC_DIR}Components/ScreenCapture.h
+    ${SRC_DIR}Components/ScreenCapture.cpp
     ${SRC_DIR}Components/Controller.h
     ${SRC_DIR}Components/Controller.cpp
     ${SRC_DIR}Components/Encoder.h
```

Comment:
should cmakelist be added in a single branch and a pr?

Reply:
If they are unrelated, then they can go in a separate PR. If they are related, they should go in the same PR.